.gitignore

```diff
@@ -5,6 +5,9 @@ target/
 !**/src/main/**/target/
 !**/src/test/**/target/
 
+# Ignore attachments
+/rev-connect-api/src/main/resources/public/attachments/**
+
```

Comment:
We are going to do these configuration commits in their own PR for sanity sake :D. For now just untrack whatever gets saved in that folder